config/locales/en_GB.yml

```diff
@@ -371,6 +371,8 @@ en_GB:
     ends_at: Ends At
     ends_on: Ends On
     name: Name
+    first_name: First name
+    last_name: Last name
```

Comment:
We don't change other locales than `en.yml` because it would lead to conflicts with our Transifex process.

Reply:
Ak ok, I just followed what was there in first commits, I will update it in Transifex then.